pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -167,7 +166,7 @@ func (s *Server) watchRootCertAndGenKeyCert(stop <-chan struct{}) {
 		select {
 		case <-stop:
 			return
-		case <-time.After(controller.NamespaceResyncPeriod):
+		case <-time.After(time.Second * 60):
```

Comment:
Why replace controller.NamespaceResyncPeriod with the hardcoded value of time.Second * 60?

Reply:
Will make it a const in the pkg